guides/dust-helpers.md

```diff
@@ -226,3 +226,17 @@ function yell(chunk, context, bodies, params) {
 }
 dust.helpers.yell = yell;
 ```
+
+## Returning a Value (Dust 2.7.1)
+
+Helpers can return a value directly. When a helper returns a value, by default the value will be HTML-sanitized. You can pass filters to a helper by adding a `filters` param.
+
+    {@myHelper value=name filters="|s|uc" /}
+
+If you need more control over the default behavior (for example if you want to render a body or not HTML-sanitize by default), your helper needs to return the Chunk it's passed instead.
```

Comment:
The last part of this sentence is a bit awkward. Maybe it could be something like "your helper needs to return the `Chunk` that was passed to it."